### ISSUE TITLE
Image opening slowdown using 1.5.0

### ISSUE THREAD

**Issue by Fironet** [poppeman/Pictus#50]
Updated to 1.5.0 but have noticed a slowdown when opening images. See the attached GIF for a comparison opening/closing the same image between 1.4.0 and 1.5.0 (apologies for the slightly large size). Unsure what has changed that may be causing it. Using the 64-bit versions.

![versions](https://cloud.githubusercontent.com/assets/6971981/11167184/8afe8c4e-8bab-11e5-86b0-55fe7331a756.gif)

Also upon launch the menus and settings are in a different language (presumably Swedish), even though the Pictus.ini still states English is the set language.

![language](https://cloud.githubusercontent.com/assets/6971981/11167186/b5e142da-8bab-11e5-9f3b-f41b5a5c1ff4.png)

![language 2](https://cloud.githubusercontent.com/assets/6971981/11167191/c9e92de2-8bab-11e5-9fed-200aeef1ed25.png)

I probably should have split the issues, which I can do if need be.

**Comment by poppeman** [comment]
Does it mainly happen on the first image on launch? Also, could you post an image that is particularly sluggish?

**Comment by poppeman** [comment]
I've released a new version that fixes the language issue at least so no need to split the issue anymore.

**Comment by Fironet** [comment]
> Does it mainly happen on the first image on launch?

The delay in rendering is consistent whether directly launching an image via double-click or by navigating once Pictus is opened, if that's what you mean.

> Also, could you post an image that is particularly sluggish?

Tested with some particularly large images (4K+) but since both 1.4 and 1.5 have a delay when opening such large images it's harder to notice. On medium sized images the difference becomes more noticeable as 1.4 renders the images more quickly.

May as well attach the image used for the GIF above, an addition to a larger image for comparison.

![tpp alt cover](https://cloud.githubusercontent.com/assets/6971981/11173592/98fe303a-8c6c-11e5-85a3-0d3455eb102d.jpg)

![gta v](https://cloud.githubusercontent.com/assets/6971981/11173593/9a73c420-8c6c-11e5-8fb8-be1515fe1ea8.jpg)

**Comment by Fironet** [comment]
I should also note some of the settings used:

Magnification: Bilinear
Minification: Lanczos3
Default zoom mode: 'Fit image'
Resize behavior: Reduce only
Reset pan between images: [Unchecked]
Resize window to fit image: Center window on screen

**Comment by poppeman** [comment]
Yeah, I accidentally left in some logging that didn't consider portable-mode. Pictus seems fairly popular with portable apps-users so that one didn't pass by silently :)

**Comment by poppeman** [comment]
I might have solved it, but the difference is not quite as obvious on my machine so it'd be great if you can test it.

https://poppeman.se/dl/pictus/1.5.2/PictInst.exe
https://poppeman.se/dl/pictus/1.5.2/pictus64.zip
https://poppeman.se/dl/pictus/1.5.2/pictus32.zip

**Comment by Fironet** [comment]
> I might have solved it, but the difference is not quite as obvious on my machine so it'd be great if you can test it.

Tested the 64-bit version, as that's the version I have of 1.4 and all others. Thought I'd record the results using OBS then extract frames every 25ms using PotPlayer to measure the difference. Hard to tell when exactly the image was clicked/launched so I measured it from when the Pictus window frame appears to when the image fully renders. Took two different samples from the video of each version.

Metal Gear Solid V parody cover image:

v1.5.2 = 5 frames.

v1.4 = 4 frames.

So a frame of difference. However the surprising results came from the larger image where v1.5.2 rendered pretty consistently at the same speed while v1.4 varied sometimes up to 26 frames (when it does this the image appears grey for a half-second, unlike v1.5.2).

Grand Theft Auto V screenshot image:

v1.5.2 = 14-15 frames.

v1.4 = 16-26 (!) frames.

So on the whole seems like an improvement in speed :thumbsup:
